homematic/CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+#### :warning: This add-on is considered to be obsolete/retired in favor of the much more advanced third-party [RaspberryMatic CCU](https://github.com/jens-maus/RaspberryMatic/tree/master/home-assistant-addon) add-on for running a HomeMatic/homematicIP smart home central within HomeAssistant. If you want to migrate to the new add-on, please make sure to update to the latest version of this old "HomeMatic CCU" add-on first and then use the WebUI-based backup routines to export a `*.sbk` config backup file which you can then restore in the new "RaspberryMatic CCU" add-on afterwards (cf. [RaspberryMatic Documentation](https://github.com/jens-maus/RaspberryMatic/wiki/Installation-HomeAssistant)) :warning:
+
+## 11.4.0
```

Comment:
```suggestion
## 99.0.0
```

Reply:
@pvizeli Why are you suggesting to set the version number to 99.0.0 ?

Reply:
Is okay now?

Reply:
it's fine, yes.